_includes/sidebar.html

```diff
@@ -9,6 +9,8 @@
       <li>
         <a href="{{folder.external_url}}" target="_blank">{{ folder.title }}</a>
       </li>
+      {% elsif folder.spacer %}
+      <br>
```

Comment:
I'm unfortunately pretty html/css illiterate :-). Got a tip for what that would look like?